packages/block-library/src/missing/test/edit-integration.native.js

```diff
@@ -1,81 +1,171 @@
 /**
  * External dependencies
  */
-import { initializeEditor, fireEvent, within } from 'test/helpers';
+import {
+	fireEvent,
+	getBlock,
+	initializeEditor,
+	screen,
+	setupCoreBlocks,
+	withFakeTimers,
+	within,
+} from 'test/helpers';
+import { Platform } from 'react-native';
 
 /**
  * WordPress dependencies
  */
-import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+import { unregisterBlockType } from '@wordpress/blocks';
 import { setLocaleData } from '@wordpress/i18n';
+import { requestUnsupportedBlockFallback } from '@wordpress/react-native-bridge';
+
+// Override modal mock to prevent unmounting it when is not visible.
+// This is required to be able to trigger onClose and onDismiss events when
+// the modal is dismissed.
+jest.mock( 'react-native-modal', () => {
+	const mockComponent = require( 'react-native/jest/mockComponent' );
+	return mockComponent( 'react-native-modal' );
+} );
 
-/**
- * Internal dependencies
- */
-import { registerCoreBlocks } from '../..';
+const TABLE_BLOCK_HTML = `<!-- wp:table -->
+<figure class="wp-block-table"><table><tbody><tr><td>1</td><td>2</td></tr><tr><td>3</td><td>4</td></tr></tbody></table></figure>
+<!-- /wp:table -->`;
+const MODAL_DISMISS_EVENT = Platform.OS === 'ios' ? 'onDismiss' : 'onModalHide';
 
-beforeAll( () => {
-	// Mock translations.
-	setLocaleData( {
-		'block title\u0004Table': [ 'Tabla' ],
-		"'%s' is not fully-supported": [ '«%s» no es totalmente compatible' ],
-	} );
+setupCoreBlocks();
 
-	// Register all core blocks.
-	registerCoreBlocks();
+beforeAll( () => {
+	// For the purpose of this test suite we consider Reusable blocks/Patterns as unsupported.
+	// For this reason we unregister it to force it to be rendered as an unsupported block.
+	unregisterBlockType( 'core/block' );
 } );
 
-afterAll( () => {
-	// Clean up translations.
-	setLocaleData( {} );
+describe( 'Unsupported block', () => {
+	describe( 'localized elements', () => {
+		beforeEach( () => {
+			// Mock translations.
+			setLocaleData( {
+				'block title\u0004Table': [ 'Tabla' ],
+				"'%s' is not fully-supported": [
+					'«%s» no es totalmente compatible',
+				],
+			} );
+		} );
+
+		afterEach( () => {
+			// Clean up translations.
+			setLocaleData( {} );
+		} );
+
+		it( 'requests translated block title in block placeholder', async () => {
+			await initializeEditor( {
+				initialHtml: TABLE_BLOCK_HTML,
+			} );
+
+			const [ missingBlock ] = await screen.findAllByLabelText(
```

Comment:
Why do we need a `AllBy` query? Are there multiples of this element?

Reply:
This code remains as it was before this PR, so we'd need to check what was the original intention when it was introduced. In any case, I think we could easily replace this with the `getBlock` test helper. I'll update it.

Reply:
Solved in https://github.com/WordPress/gutenberg/pull/55133/commits/93f452832ba03d94debec9d479e1f79279524a96.